gradle.properties

```diff
@@ -6,3 +6,4 @@ kotlinVersion=1.3.70
 org.gradle.jvmargs=-Xmx3g -XX:MaxPermSize=2048m -XX:+HeapDumpOnOutOfMemoryError
 org.gradle.parallel=true
 org.gradle.caching=true
+springVersion=5.3.0-SNAPSHOT
```

Comment:
I will also have to update ```ext.reactorVersion = "Dysprosium-SR+"``` because spring-tx is using ```ext.reactorVersion = "2020.0.0-SNAPSHOT"```. If I don't do that, test are failing with ```NoSuchMethodError``` in ```org.springframework.transaction.reactive.TransactionContextManager.currentContext()```

Reply:
I will also have to update ```ext.reactorVersion = "Dysprosium-SR+"``` because spring-tx is using ```ext.reactorVersion = "2020.0.0-SNAPSHOT"```. If I don't do that, test are failing with ```NoSuchMethodError``` in ```org.springframework.transaction.reactive.TransactionContextManager.currentContext()``` which is using ```Mono.deferContextual``` which is not available with Dysprosium-SR+ release train